### PR TITLE
Feature/query microservices

### PR DESCRIPTION
**Pull request by jwomeara**

### CODE REVIEW

File: api/src/main/java/datawave/webservice/response/objects/KeyBase.java [hunk 1]
```diff
@@ -6,11 +6,8 @@
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlSeeAlso;
 
-import com.fasterxml.jackson.annotation.JsonTypeInfo;
-
 import datawave.webservice.query.result.event.HasMarkings;
 
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY)
```
Comment (jwomeara) [inline api/src/main/java/datawave/webservice/response/objects/KeyBase.java:13]: This was removed to try and fix enunciate issues, but I think the query microservice PRs disable enunciate...  Need to look into this.

**Review by jwomeara — COMMENTED** [review]
Notable changes in this PR:
 - Major version bumps

**Review by apmoriarty — APPROVED** [review]

**Review by matthpeterson — APPROVED** [review]

File: api/src/main/java/datawave/webservice/response/objects/KeyBase.java (hunk 1 above)
Comment (alerman) [reply api/src/main/java/datawave/webservice/response/objects/KeyBase.java:13]: I wouldnt be opposed to removing enunciate in favor of just about anything else

**Review by alerman — APPROVED** [review]

**Review by avgAGB — APPROVED** [review]

File: api/src/main/java/datawave/webservice/response/objects/KeyBase.java (hunk 1 above)
Comment (jwomeara) [reply api/src/main/java/datawave/webservice/response/objects/KeyBase.java:13]: yeah, we recently removed it from integration, so im not going to worry about it.

**Review by keith-ratcliffe — APPROVED** [review]

**Pull request merged**